src/IssueViz.Security.UnitTests/OpenInIDE/Http/OwinPipelineProcessorTests.cs

```diff
@@ -35,16 +36,52 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests.OpenInIDE
     [TestClass]
     public class OwinPipelineProcessorTests
     {
-        const string Origin = "http://origin";
+        private const string Origin = "http://origin";
 
         [TestMethod]
         public void MefCtor_CheckIsExported()
         {
+            // IOwinRequestHandlers are ImportMany -> optional
             MefTestHelpers.CheckTypeCanBeImported<OwinPipelineProcessor, IOwinPipelineProcessor>(
-                MefTestHelpers.CreateExport<IEnumerable<IOwinPathRequestHandler>>(Array.Empty<IOwinPathRequestHandler>()),
                 MefTestHelpers.CreateExport<ILogger>());
         }
 
+        [TestMethod]
+        public void MefCtor_CheckIsExported_RequestHandlersAreImported()
```

Comment:
New test to check that the `ImportMany` request handlers are imported as expected.
This test is logically the same as `MefCtor_CheckIsExported_LocationStoresAreImported` (below).